tests/test_C0_data_processing.py

```diff
@@ -1080,7 +1084,36 @@ def test_change_sign_of_feedin_tariff_zero(caplog):
     ), f"A msg is logged although the feed-in tariff is not changed."
 
 
+def test_compute_timeseries_properties_TIMESERIES_in_dict_asset():
```

Comment:
This belongs to #724 - can you remove the test from this PR to keep it clean? @mahendrark please help if @paragpatil39 needs it.

Reply:
If (hopefully) you added this in a single commit it is easy to fix :)